pynicotine/gtkgui/dialogs/fastconfigure.py

```diff
@@ -172,34 +159,25 @@ def on_add_shared_folder(self, *_args):
 
     def on_edit_shared_folder_response(self, dialog, _response_id, iterator):
 
-        virtual_name = dialog.get_entry_value()
+        new_virtual_name = dialog.get_entry_value()
 
-        if not virtual_name:
+        if not new_virtual_name:
             return
```

Comment:
A noop is undesirable here.

Consider returning the virtual_name to the default instead (i.e. based on the folder name) in this case whereupon the text entry is cleared to `""` then pass `None`.

Reply:
Is it better now?

Reply:
That is the correct dialog behaviour yes.